public/docs/_examples/user-input/dart/lib/todo_list.dart

```diff
@@ -0,0 +1,41 @@
+library user_input.todo_list;
+
+import 'dart:html';
+
+import 'package:angular2/angular2.dart';
+
+@Component(selector: 'todo-list')
+@View(
```

Comment:
Yep, it started out with @View being required, but now @View shouldn't be required unless you have multiple views (which might not even be possible yet).